File: appserver-legacy/src/main/java/org/radarbase/fcm/downstream/DisabledFcmSender.java
package org.radarbase.fcm.downstream;

import org.radarbase.fcm.model.FcmDownstreamMessage;

public class DisabledFcmSender implements FcmSender {
    @Override
    public void send(FcmDownstreamMessage message) throws Exception {
        // do nothing
    }

    @Override
    public boolean doesProvideDeliveryReceipt() {
        return false;
    }
}

File: src/main/java/org/radarbase/fcm/downstream/DisabledFcmSender.kt
package org.radarbase.fcm.downstream

import org.radarbase.fcm.model.FcmDownstreamMessage

class DisabledFcmSender : FcmSender {
    @Throws(Exception::class)
    override fun send(message: FcmDownstreamMessage) {
        // do nothing
    }

    override fun doesProvideDeliveryReceipt(): Boolean {
        return false
    }
}
